scripts/valhalla_build_config

```diff
@@ -36,6 +36,7 @@ config = {
     'include_bicycle': True,
     'include_pedestrian': True,
     'include_driving': True,
+    'include_osmids': False,
```

Comment:
nit: we need to include the word "node" in here. we already include the osm way ids for all the ways but i think its not descriptive enough to just say "osmid"

Reply:
Agreed.

Reply:
Changed to `include_osmids_for_nodes`.